### ISSUE TITLE
Stream Interrupted

### ISSUE THREAD

**Issue by RaddMike** [TwitchRecover/TwitchRecover#10]
Hi,

I used the recover tool to successfully regain the first part of my stream that was deleted.

During the stream I had to stop and restart the stream for a very brief moment - Twitchtracker still views the stream as one long stream.

However, I was only able to regain the first part of my stream and cannot seem to find the second part even though I re-entered the precise timestamp that it would have been at. And it's supposed to be only one stream.

Any thoughts on how I can resolve this?

Thanks!

**Comment by TCarneiro91** [comment]
If Sully Gnone deteccted that you started a 2nd stream, it''s possible to get the 2nd Vod id and make a brute force scan to the minute

**Comment by RaddMike** [comment]
> If Sully Gnone deteccted that you started a 2nd stream, it''s possible to get the 2nd Vod id and make a brute force scan to the minute

Thank you for your reply Tcarneiro91. The issue is it says it's only 1 stream but when I used the tool I only obtained the first part of the stream just before I had to reboot it.

**Comment by TCarneiro91** [comment]
What's your channel?

**Comment by RaddMike** [comment]
> What's your channel?

tacticotwitch

**Comment by TCarneiro91** [comment]
yeah, sully gnome also detects as one complete stream, maybe there's no solution from me, maybe the owner can help you

**Comment by daylamtayari** [comment]
@RaddMike Here is the list of the M3U8 links for the second part of that VOD:   
```
https://vod-secure.twitch.tv/20f089acb73da8049da1_tacticotwitch_40387775116_1608249493/chunked/index-dvr.m3u8  
https://vod-metro.twitch.tv/20f089acb73da8049da1_tacticotwitch_40387775116_1608249493/chunked/index-dvr.m3u8
https://dqrpb9wgowsf5.cloudfront.net/20f089acb73da8049da1_tacticotwitch_40387775116_1608249493/chunked/index-dvr.m3u8
https://ds0h3roq6wcgc.cloudfront.net/20f089acb73da8049da1_tacticotwitch_40387775116_1608249493/chunked/index-dvr.m3u8
https://dqrpb9wgowsf5.cloudfront.net/20f089acb73da8049da1_tacticotwitch_40387775116_1608249493/chunked/index-dvr.m3u8
```

**Comment by TCarneiro91** [comment]
@daylamtayari Is that from the 2.0 version? How did you got that?

**Comment by daylamtayari** [comment]
For clarification on how I retrieved them, when you restart a stream instantly, some tracking sites like Twitch Tracker or Sully Gnome won't detect the interruption in the stream.  
However, I personally find Streamscharts, another similar website to be better at tracking stream interruptions and restarts (they will be directly integrated into the 2.0 version).   
So in this case, I went on streamscharts, found that they had indeed detected the stream interruption and then got the stream ID from the website and exact stream time from their API and then here, inputted the values manually and got these links.   
All of this will be automatically integrated in the 2.0 version where you would only need to paste the Streamscharts stream link and it retrieves all the data automatically.
